grails-app/conf/BuildConfig.groovy

```diff
@@ -24,7 +24,7 @@ grails.project.test.reports.dir = "target/test-reports"
 def dm, dmClass
 try {
     dmClass = new GroovyClassLoader().parseClass(
-            new File('../transmart-dev/DependencyManagement.groovy'))
+            new File('../transmart-dev/SmartRDependencyManagement.groovy'))
```

Comment:
No I don't think so.
Actually, as I understand `transmart-dev/DependencyManagement.groovy`, it's a file used to override `transmartApp/grails-app/conf/BuildConfig.groovy` in order for local developments to work without modifying transmartApp source code.
For example here is the one used for building our Sanofi instance using Docker: https://github.com/anthony-o/transmart-dev/blob/sanofi-release-16.1/DependencyManagement.groovy
As you can see, I can override local projects & `transmartApp` dependencies.
So for this patch, I modified the local override file from `DependencyManagement.groovy` to `SmartRDependencyManagement.groovy` in order to have a specific override file for SmartR, and don't use the one which I understand is used for building parent transmartApp project.
At least, with this config I have no problem to build & use [our specific version of SmartR](https://github.com/anthony-o/SmartR/tree/sanofi-release-16.1) (which for now has only the patch I've submitted here) using [our Docker project](https://github.com/anthony-o/transmart-docker/tree/sanofi-release-16.1).